src/SimpleGraphs/generators/randgraphs.jl

```diff
@@ -22,13 +19,16 @@ julia> SimpleGraph(5, 7)
 {5, 7} undirected simple Int64 graph
 ```
 """
-function SimpleGraph{T}(nv::Integer, ne::Integer; seed::Int=-1) where T <: Integer
+function SimpleGraph{T}(
+    nv::Integer, ne::Integer;
+    rng::Union{Nothing, AbstractRNG}=nothing, seed::Union{Nothing, Integer}=-1
+) where T <: Integer
     tnv = T(nv)
     maxe = div(Int(nv) * (nv - 1), 2)
     @assert(ne <= maxe, "Maximum number of edges for this graph is $maxe")
-    ne > div((2 * maxe), 3)  && return complement(SimpleGraph(tnv, maxe - ne, seed=seed))
+    ne > div((2 * maxe), 3)  && return complement(SimpleGraph(tnv, maxe - ne, rng=rng, seed=seed))
 
-    rng = getRNG(seed)
+    rng = rng_from_rng_or_seed(rng, seed)
```

Comment:
If we switch this with the line above, we can pass this  to the `SimpleGraph` call on the line above.

Reply:
Good point, fixed.